capstone/src/main/webapp/script.js

```diff
@@ -25,6 +25,7 @@ let _map;
 let _center;
 let _showSmallBusiness = false;
 let _showBlackOwnedBusiness = false;
+let _keyword;
```

Comment:
nit: I think it would be great to add comments above these describing their purpose in the file.